docs/variants/qemu_q35/development.md

```diff
@@ -0,0 +1,138 @@
+# Dasharo (UEFI) v0.1.0 for QEMU Q35
+
+## Intro
+
+1. Please check your qemu setup and launch qemu outside
+docker instance (host machine) using built OVMF firmware image.
+
++ QEMU Installation:
+    - [Install QEMU on your linux distro](<https://www.qemu.org/download/#linux>)
+
+ ```bash
+ qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd -debugcon file:debug.log -global isa-debugcon.iobase=0x402 -global ICH9-LPC.disable_s3=1 -net none -machine q35,smm=on
+ ```
+
++ `-drive` indicate device is pflash with firmware image of built OVMF.fd image.
+
++ `-debugcon file:debug.log -global isa-debugcon.iobase=0x402`
+default OVMF build writes debug messages to IO port **0x402**.
+The following qemu command line options save them in the file called debug.log.
+
++ `-global ICH9-LPC.disable_s3=1` SMM is put to use in the S3 suspend
```

Comment:
I think for S3 would be good to have it enabled.

Reply:
I guess this is not that simple, I had problems booting with S3 enabled. Question is how S3 works in QEMU. In general this seem to be some workaround which I was not able to decode why it is needed. If we would figure out we should document that and probably propagate to OST2 Arch4021.

More to that at some point we have to align Dasharo with OST2, but this is other topic and longer term action. Maybe we can do that as part of Arch4221 UEFI Secure Boot